plugins/grafana-custom-plugins/grafana-dependency-plugin/package.json

```diff
@@ -3,27 +3,72 @@
   "version": "1.0.0",
   "description": "service dependency graph",
   "scripts": {
-    "build": "grafana-toolkit plugin:build",
-    "test": "grafana-toolkit plugin:test",
-    "dev": "grafana-toolkit plugin:dev",
-    "watch": "grafana-toolkit plugin:dev --watch",
-    "sign": "grafana-toolkit plugin:sign",
-    "start": "yarn watch"
+    "build": "webpack -c ./webpack.config.ts --env production",
+    "dev": "webpack -w -c ./webpack.config.ts --env development",
+    "e2e": "yarn exec cypress install && yarn exec grafana-e2e run",
```

Comment:
Maybe you can follow this PR https://github.com/antrea-io/theia/pull/587 to remove some unused scripts and dependencies in this PR to avoid some `Vulnerability alerts` bumped up after this PR is merged.
Note:
Not all dependencies detected by [depcheck](https://www.npmjs.com/package/depcheck) is unused.